SignalServiceKit/src/Messages/DeviceSyncing/OWSIncomingSentMessageTranscript.m

```diff
@@ -47,7 +47,7 @@ - (instancetype)initWithProto:(OWSSignalServiceProtosSyncMessageSent *)sentProto
 
     _quotedMessage =
         [TSQuotedMessage quotedMessageForDataMessage:_dataMessage thread:_thread relay:relay transaction:transaction];
-    _contact = [OWSContacts contactForDataMessage:_dataMessage relay:relay transaction:transaction];
+    _contactShare = [OWSContactConversion contactShareForDataMessage:_dataMessage relay:relay transaction:transaction];
```

Comment:
much better!

Reply:
There's more renaming to be done - as you pointed out, we could use contactShareVM in a lot of places now.